src/tools/crossgen2/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2689,6 +2689,13 @@ private void MethodCompileComplete(CORINFO_METHOD_STRUCT_* methHnd)
             return null;
         }
 
+        private bool getTailCallHelp(CORINFO_METHOD_STRUCT_* hTarget, CORINFO_SIG_INFO* callSiteSig, CORINFO_GET_TAILCALL_HELP_FLAGS fLAGS, ref CORINFO_TAILCALL_HELP pResult)
+        {
+            // Slow tailcalls are not supported yet
+            // https://github.com/dotnet/corert/issues/1683
+            return false;
+        }
+
```

Comment:
@jkotas The `getTailCallCopyArgsThunk` above is not throwing exception. Does that need to be fixed?

Reply:
Yes, it would be nice to fix. I suspect that it is not even called because of the slow tailcalls get disabled earlier.

Does the JIT still use `getTailCallCopyArgsThunk` anywhere with these changes?

Reply:
No, it can be removed with this change or after. x86 does not use the copy arg thunks. I have tried to do it once but there are many places in the code base that has tailcall helper code and I was not too sure that I would find it all.